akka-stream/src/main/boilerplate/akka/stream/scaladsl/SourceApply.scala.template

```diff
@@ -7,6 +7,12 @@ import akka.stream.{ Outlet, Shape, Graph }
 
 trait SourceApply {
 
+  def apply2[Out,Mat](buildBlock: FlowGraph.Builder[Mat] => Outlet[Out]): Source[Out, Mat] = {
```

Comment:
Initially I looked for apply with [Out,Mat] instead of [Out,Unit] but @drewhk gave me a hint that Source.wrap will work here